server/controller/feedsController.js

```diff
@@ -5,7 +5,10 @@ class FeedsController {
   static async getAllFeeds(req, res) {
     try {
       const userId = req.user.userId;
-      const feeds = await pool.query(queries.getAllFeeds, [userId]);
+      const { offset } = req.query;
+      const { limit } = req.query;
+      const values = [userId, limit, offset];
```

Comment:
set default for the limit and offset
`const values = [userId, limit || 10, offset || 0]`

Reply:
Alright sir. Done